Trn 358 custom theme

Kartik9400 opened the pull request:
Created a subtheme with barrio basetheme
Designed its frontpage

Kartik9400 committed:
TRN-358-custom-theme
Added a subtheme with base theme as barrio theme
Designed its front page

Kartik9400 committed:
TRN-358 Removed subtheme subfolder and libraries
As libraries and subtheme subfolder are not contribution to the created subtheme

kkmitra commented on web/themes/custom/barrio_subtheme_create.sh:
```diff
@@ -0,0 +1,9 @@
+export CUSTOM_BARRIO=innonews_custom_barrio # change this to your custom theme_name
```
No need to push these file. Please remove it.

kkmitra commented on web/themes/custom/innonews_custom_barrio/config/schema/innonews_custom_barrio.schema.yml:
```diff
@@ -0,0 +1,5 @@
+
+innonews_custom_barrio.settings:
+  type: bootstrap_barrio.settings
+  label: 'Bootstrap Barrio settings'
```
I think we can change the label here.

kkmitra reviewed (changes requested):
Looks good, just need a few changes.

Kartik9400 committed:
TRN-358
Removed barrio_Subtheme_create.sh because it is not needed
Changed color and label in theme
